src/backend/distributed/planner/multi_explain.c

```diff
@@ -108,17 +109,30 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 	instr_time planDuration;
 	Query *originalQuery = NULL;
 	RelationRestrictionContext *restrictionContext = NULL;
-
-	/* if local query, run the standard explain and return */
 	bool localQuery = !NeedsDistributedPlanning(query);
+	int cursorOptions = 0;
+
+#if PG_VERSION_NUM >= 90600
+
+	/*
+	 * Allow parallel plans in 9.6+ unless selecting into a table.
+	 * Without this, we're breaking explain for non-Citus plans.
+	 */
+	if (!into)
+	{
+		cursorOptions |= CURSOR_OPT_PARALLEL_OK;
+	}
+#endif
```

Comment:
To avoid repetitions of this, we really should add a make target that, if the postgres source is available, runs postgres' installcheck-parallel against a postgres installation with citus in shared_preload_libraries. Hm, let's just open an issue: #877